lib/widgets/draggable_scrollable_modal_bottom_sheet.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/material.dart';
+
+class _DraggableScrollableLayer extends StatelessWidget {
+  const _DraggableScrollableLayer({required this.builder});
+
+  final WidgetBuilder builder;
+
+  @override
+  Widget build(BuildContext context) {
+    return DraggableScrollableSheet(
+        // Enough less than 1 as to not extend into the device's top inset
+        maxChildSize: 0.75,
```

Comment:
If you open the sheet, then drag it lightly down, it just moves down with your finger and then stops there, still open. That feels odd ­— it should go all the way closed, I think.

It looks like one can fix this by setting `initialChildSize` slightly larger than `minChildSize`.

"Slightly", because if you set them *equal* then there's some odd other behavior: you try to drag it closed, and get the glowing overscroll indicator. (At least on Android.) But e.g. `initialChildSize: 0.26`, with the default `minChildSize` of 0.25, fixes that.

Reply:
Here's a related upstream issue, to watch out for when setting these `DraggableScrollableSheet` parameters:
https://github.com/flutter/flutter/issues/116982

Reply:
There's also this messy thread with people describing several different issues:
https://github.com/flutter/flutter/issues/36283#issuecomment-1110613055

Reply:
Yeah, while I was experimenting with `snap: true`, ISTR noticing a behavior like the one described in https://github.com/flutter/flutter/issues/116982.